controllers/usersignup/usersignup_controller.go

```diff
@@ -361,9 +361,12 @@ func (r *Reconciler) checkIfMurAlreadyExists(
 			if err != nil {
 				return true, r.wrapErrorWithStatusUpdate(ctx, userSignup, r.setStatusFailedToCreateSpace, err, "error creating Space")
 			} else if created {
+				err = r.updateStatusHomeSpace(ctx, userSignup, space.Name)
+				if err != nil {
+					return true, err
+				}
```

Comment:
the `updateStatusHomeSpace` should be called not only when a space is created, but also when it is already present